pom.xml

```diff
@@ -50,8 +50,8 @@
         <apache.httpclient.version>4.5.2</apache.httpclient.version>
         <bouncycastle.bcpkix.version>1.55</bouncycastle.bcpkix.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <jersey.version>2.25</jersey.version>
-        <jetty.version>9.2.19.v20160908</jetty.version>
+        <jersey.version>2.26-b06</jersey.version>
+        <jetty.version>9.4.6.v20170531</jetty.version>
```

Comment:
We've held off on updating to 9.3+ because they remove Java 7 support and, unfortunately, there are quite a few folks still targeting JDK7. However, it looks like `9.2.22.v20170606` would be an updated version that would work here.

I'm also open to some maven profile magic that would allow people to build against newer versions if they're fine with a JDK8 requirement.